ProblemM.cpp

```diff
@@ -1,8 +1,7 @@
 #include <iostream>
 using namespace std;
 
-int main()
-{
+int main() {
     char ch;
     cin >> ch;
 
```

Comment:
Totally not important, but:

(1) You could move `ch >= 65 && ch <= 90` to a function `isCapitalCase`

(2) Google "Magic numbers". You could create `const int CAPITAL_A_ASCII = 65` & then use it.

(3) Even better, you could: `const int CAPITAL_A_ASCII = (int) 'A'`